app/app.go

```diff
@@ -619,6 +629,17 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(clientCtx, app.BaseApp.GRPCQueryRouter(), app.interfaceRegistry, nil)
 }
 
+func (app *App) setPostHanders() {
+	postHandler, err := posthandler.NewPostHandler(
```

Comment:
Is there a reason why we are creating a new function for setting the `postHandler` and not for setting the `anteHandler`?

Reply:
I _think_ we are, is this what you mean? https://github.com/celestiaorg/celestia-app/blob/82393670fd39e2a01a13811ceaf9ffae200651f7/app/app.go#L506-L507

we don't define a method as baseapp already has one iirc

Reply:
Ah okey, got you :+1: